components/FormStyles/QuestionText.tsx

```diff
@@ -2,10 +2,51 @@ import styled from 'styled-components'
 
 export const QuestionText = styled.label`
   font-family: Source Sans Pro;
-  font-style: normal;
-  font-weight: normal;
-  font-size: 26px;
+  font-size: 16px;
   line-height: 33px;
+`
+export const AnswerText = styled.label`
+  font-family: Source Sans Pro;
+  font-size: 16px;
+  line-height: 33px;
+  font-weight: 600;
+`
 
-  color: #3d3d3d;
+const QuoteBlockBar = styled.div`
+  width: 5px;
+  background: #75ba39;
+  border-radius: 5px;
+  margin-right: 12px;
 `
+
+interface QuestionTextProps {
+  questionText: string
+}
+
+const BlockQuote: React.FC<QuestionTextProps> = ({ questionText }) => {
+  return (
+    <div style={{ display: 'flex', padding: '10px, 0px, 10px, 0px' }}>
+      <QuoteBlockBar />
+      <QuestionText>{questionText}</QuestionText>
+    </div>
+  )
+}
+
+export const QuestionsWithBlockText: React.FC<QuestionTextProps> = ({
```

Comment:
logic here is to essentially break up any questions with quotes in them, and the quotes should be displayed, so every odd numbered section of the split should be between two quotes